buzsaki_lab_to_nwb/valero/behaviorinterface.py

```diff
@@ -170,17 +168,24 @@ def run_conversion(self, nwbfile: NWBFile, metadata: dict, stub_test: bool = Fal
             "Estimated using `https://github.com/buzsakilab/buzcode/tree/master/detectors/detectStates/SleepScoreMaster`"
         )
 
-        description_of_available_states = {state: description_of_states[state] for state in available_states}
+        description_of_available_states = {state: description_of_states[state] for state in sleep_intervals}
         description = f"Description of states : {json.dumps(description_of_available_states, indent=4)}"
-        table = TimeIntervals(name="SleepStates", description=description)
-        table.add_column(name="label", description="Sleep state.")
 
         table_rows = []
         for state_name, state_intervals in sleep_intervals.items():
-            for start_time, stop_time in state_intervals:
+            if state_intervals.ndim > 1:
+                for start_time, stop_time in state_intervals:
+                    row_as_dict = dict(start_time=float(start_time), stop_time=float(stop_time), label=state_name)
+                    table_rows.append(row_as_dict)
+            else:  # This is necessary because `read_mat` returns a 1D array if there is only one interval
+                start_time, stop_time = state_intervals
                 row_as_dict = dict(start_time=float(start_time), stop_time=float(stop_time), label=state_name)
                 table_rows.append(row_as_dict)
 
+        time_intervals = TimeIntervals(name="SleepStates", description=description)
+        time_intervals.add_column(name="label", description="Sleep state.")
         sorted_table = sorted(table_rows, key=lambda x: (x["start_time"], x["stop_time"]))
-        [table.add_row(**row_as_dict) for row_as_dict in sorted_table]
-        processing_module.add(table)
+        [time_intervals.add_row(**row_as_dict) for row_as_dict in sorted_table]
+
+        processing_module = get_module(nwbfile=nwbfile, name="behavior")
```

Comment:
```suggestion
        processing_module = get_module(nwbfile=nwbfile, name="behavior", description="Contains processed behavioral data.")
```

Or `"Contains behavioral data concerning classified states."` (currently what it is on the Huszar side)

from past references https://github.com/search?q=repo%3Acatalystneuro%2Fbuzsaki-lab-to-nwb%20get_module&type=code

Should probably have a set of defaults like that stored and set in `get_module` depending on the `name`

Reply:
Added something along those lines.